### PR TITLE
Migrated Expression Resolver test.

### PR DESCRIPTION
**Pull request by cypo721**
Created ELResolver testcase.

### CODE REVIEW

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/tools/ELResolverTestCase.java [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+        String expected = "server-one=" + VALUE + "\nserver-two=default_value\n";
+        assertEquals(expected, output);
+
+        goToServerProperties("server-one");
+        removeProperty();
+    }
+
+
+
+    /* UTILS */
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/tools/ELResolverTestCase.java]: I suggest to extract all UTILS methods to UI abstraction class .. something like `ServerConfigurationPage` to separate the UI and test logic.

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/tools/ELResolverTestCase.java [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    private void goToServerProperties(String server) {
+        FinderNavigation navigation = new FinderNavigation(browser, DomainRuntimeEntryPoint.class)
+                .addAddress(FinderNames.BROWSE_DOMAIN_BY, FinderNames.HOSTS)
+                .addAddress(FinderNames.HOST, "master")
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/tools/ELResolverTestCase.java]: Please replace `"master"` with `org.jboss.hal.testsuite.util.ConfigUtils.getDefaultHost()`

**Comment by cypo721** [comment]
Updated.

**Comment by zebrik** [comment]
Thank you.

**Pull request merged**